core/src/i18n/pl.yaml

```diff
@@ -59,8 +56,6 @@ cron-jobs:
   title: Cron Jobs
 custom-resource-definitions:
   title: Definicje zasobów niestandardowych
-deployments:
-  title: Deploymenty
 discovery-and-network:
   title: Odkrywanie i Sieć
 functions:
```

Comment:
Re line 86: do we capitalize `Przestrzeń Nazw` in Polish as well? 
If so, please captalize it in the whole file (and then you should also capitalize all the other resource names that are capitalized in English).
Otherwise, make it lowercase here.

```
    back: Powrót do przestrzeni nazw
```

❗ Following this, `sonda` in line 77 should also be changed to lowercase.